bigquery/browse_table.rb

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env ruby
+
+# Copyright 2015 Google, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START all]
+require "gcloud"
+# A short sample demonstrating browsing a BigQuery table
+# This uses Application Default Credentials to authenticate.
+# @see https://cloud.google.com/bigquery/bigquery-api-quickstart
+def browse_table project_id, dataset_id, table_id, max=10
+  # [START browse_table]
+  gcloud = Gcloud.new project_id
+  bigquery = gcloud.bigquery
+
+  dataset = bigquery.dataset dataset_id
+  table = dataset.table table_id
+  row_num = 0
+  page_token = nil
+  loop do
+    data = table.data :token => page_token, :max => max
```

Comment:
FYI

```ruby
data = table.data token: page_token, max: max
```

Don't use hash rockets `=>` unless the key of the hash must be a String or type besides a Symbol ... in those cases, use of a hash literal might not be recommended